<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>display</class>
 <widget class="QWidget" name="display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="show_data">
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_mask">
     <property name="text">
      <string>Mask</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_massif">
     <property name="text">
      <string>Massif</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_points">
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_contour">
     <property name="text">
      <string>Contour</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="show_solidangle">
     <property name="text">
      <string>Solid angle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
